data_splitting.py

```diff
@@ -114,21 +117,20 @@ def process_data_splitting_scenario(x_train, y_train, x_test, y_test, dataset_na
         y_node_train = y_train[train_idx,]
         
         # Test data
-        if testset_option == 'Split':
+        if scenario.testset_option == 'Split':
             x_node_test = x_test[test_idx]
             y_node_test = y_test[test_idx]
-        elif testset_option == 'Global':
+        elif scenario.testset_option == 'Global':
             x_node_test = x_test
             y_node_test = y_test
         else:
-            print('This testset_option scenario is not recognized. Quitting with quit()')
-            quit()
-        
+            raise NameError('This testset_option scenario is not recognized. Quitting with quit()')
```

Comment:
You should remove "Quitting with quit()" for coherence here :)

Reply:
will do